back-office/src/pages/ProfilScreen/ProfilScreen.jsx

```diff
@@ -0,0 +1,38 @@
+//@flow
+import React from 'react'
+
+import './ProfilScreen.css'
+import { User } from '../../user';
+import { NewUserLine } from '../../components/user'
+
+class ProfilScreen extends React.Component {
+  
+  props: {
+    user: User,
+    updateUser: () => void,
+    updateUserProperty: () => void,
+  }
+
+  handleSave = (user) => {
+    this.props.updateUser(user)
+  }
+
+  handleChange = (name, value) => {
+    this.props.updateUserProperty(name, value)
+  }
+
```

Comment:
Same thing here, you can have a Functional component and move the `handleChange` and `handleSave` definition to the container